sdk/search/Microsoft.Azure.Search.Common/src/Microsoft.Azure.Search.Common.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Common types needed by the Azure Search .NET libraries. This is not the package you are looking for; It is only meant to be used as a dependency.</Description>
     <AssemblyTitle>Microsoft Azure Search Common Library</AssemblyTitle>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <VersionPrefix>9.0.1</VersionPrefix>
+    <VersionPrefix>9.0.2</VersionPrefix>
```

Comment:
nit: Might be worth moving the version property into a common props file to remove some redundancy.

Reply:
Sure, we'll do that in a subsequent PR